CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,296 @@
+        template](https://github.com/open-feature/community/ISSUE_TEMPLATE/membership.md))
+   * Make sure that the list of contributions included is representative of your
+        work on the project.
+ 3. Have your sponsoring reviewers reply confirmation of sponsorship: `I support`
+ 4. Once your sponsors have responded, your request will be reviewed by the
+   Technical Committee (TC).  Any TC member can review the requirements and add
+   Members to the GitHub org.
+
+
+### Triager
+Triagers assist the maintainers and approvers with project management and
+backlog organization. The specific workflows and triage requirements depend on
+the project, and are set by the project maintainers.
+
+Defined by: [Triage permissions](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level),
+with the names of the current Triagers commited to git, either in CONTRIBUTING,
+CODEOWNERS, or the bottom of the README.
+
+Triagers may be code contributors, but writing code is not a requirement for
+becoming a triager. Triagers are encouraged to be active participants in project
+meetings, chat rooms, and other discussion forums.
+
+* Requirements
+
+- Nominated by a maintainer, with no objections from other maintainers.
+- Consistently attend meetings and interact with issues.
+
+* Responsibilities and privileges
+
+- Have an understanding of the goals and workflows defined by the maintainers.
+- Respond to new PRs and Issues by asking clarifying questions.
+- Organize the backlog by applying labels, milestones, assignees, and projects.
+
+
+The process of becoming a Trager is:
+
+   1. The contributor is nominated by opening a PR against the appropriate repository, which adds their GitHub username to the respective GitHub team for one or more directories.
+   2. At least two members of the team that owns that repository or main directory, who are already Approvers, approve the PR.
+
+## Approver
+
+Code approvers are able to both review and approve code contributions, as well
+as help maintainers triage issues and do project management.
+
+While code review is focused on code quality and correctness, approval is
+focused on holistic acceptance of a contribution including: backwards/forwards
+compatibility, adhering to API and flag conventions, subtle performance and
+correctness issues, interactions with other parts of the system, etc.
+
+Defined by: [CODEOWNERS
+workflow](https://help.github.com/en/articles/about-code-owners).
+
+Approver status can be scoped to a part of the codebase. For example, critical
+core components may have higher bar for becoming an approver.
+
+* Requirements
+
+The following apply to the part of the codebase for which one would be an
+approver in the `CODEOWNERS` files.
+
+   * Reviewer of the codebase for at least 1 month
+   * Reviewer for or author of at least 10 substantial PRs to the codebase,
```

Comment:
I think 10 is a bit specific and arbitrary. I think I'd like to be a little less prescriptive here.